packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox-group/example.html

```diff
@@ -0,0 +1,331 @@
+        type="checkbox"
+        name="topping-1"
+        value="sprinkles"
+        checked
+      />
+      <label slot="label">Sprinkles</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="topping-1"
+        value="caramelSauce"
+      />
+      <label slot="label">Caramel Sauce</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="topping-1"
+        value="cherry"
+        disabled
+      />
+      <label slot="label">Cherry</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="topping-1"
+        value="strawberries"
+      />
+      <label slot="label">Strawberries</label>
+      <span slot="error">This is an error message</span>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="topping-1"
+        value="cookieCrumble"
+      />
+      <label slot="label">Cookie Crumble</label>
+      <span slot="help">This is a help message</span>
+    </gux-form-field-checkbox>
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<h2>Example with Group Error</h2>
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-form-field-checkbox-group-beta>
+    <label slot="group-label">Toppings</label>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="topping-1" value="hotFudge" />
+      <label slot="label">Hot Fudge</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="topping-1"
+        value="cherry"
+        disabled
+      />
+      <label slot="label">Cherry</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="topping-1"
+        value="strawberries"
+      />
+      <label slot="label">Strawberries</label>
+      <span slot="error">This is an error message</span>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="topping-1"
+        value="cookieCrumble"
+      />
+      <label slot="label">Cookie Crumble</label>
+      <span slot="help">This is a help message</span>
+    </gux-form-field-checkbox>
+    <span slot="group-error">This is an error message</span>
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<h2>Example with Group Help text</h2>
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-form-field-checkbox-group-beta>
+    <label slot="group-label">Toppings</label>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="topping-1" value="hotFudge" />
+      <label slot=" label">Hot Fudge</label>
```

Comment:
Minor typo, need to remove space in `slot=" label"`. I think there is one more instance of this down further also.